scripts/base/init-bare.bro

```diff
@@ -3710,11 +3710,36 @@ export {
 	## external harness and shouldn't output anything to the console.
 	const errors_to_stderr = T &redef;
 }
+
+module Fanout;
+
+type Method: enum {
```

Comment:
Is it really helpful to expose all these methods? Bro requires seeing all packets that are part of the same flow, which I believe means that only the hashing will really work for it, no?